app/views/widgets/right_column/_picasa_gallery.html

```diff
@@ -2,7 +2,7 @@
 	<h3>Picasa Widget</h3>
     <ul>
     </ul>
-    <a class="more-link" href="javascript:;">
-              See Full Gallery <span class="arrow">»</span>
-            </a>
+    <a class="more-link" href="https://get.google.com/albumarchive/105740368086394902806">
```

Comment:
The link we were building was broken. (I don't think Picasa even exists anymore. It's now Google Photos or Google Album or something.)

Any reason why we don't just hardcode the link here?